src/PIL/features.py

```diff
@@ -230,6 +230,9 @@ def pilinfo(out=None, supported_formats=True):
     """
     Prints information about this installation of Pillow.
     This function can be called with ``python3 -m PIL``.
+    It can also be called with ``python3 -m PIL --report`` to have
+    "supported_formats" set to ``False``, omitting the list of all supported image
+    file formats.
```

Comment:
```suggestion
    It can also be called with ``python3 -m PIL.report`` or ``python3 -m PIL --report``
    to have "supported_formats" set to ``False``, omitting the list of all supported
    image file formats.
```